src/views/VenuesListWithFilter/FilterList/Amenities/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import ListItemButton from '@mui/material/ListItemButton'
+import ListItemIcon from '@mui/material/ListItemIcon'
+import ListItemText from '@mui/material/ListItemText'
+import ListItem from '@mui/material/ListItem'
+import Checkbox from '@mui/material/Checkbox'
+
+export function Amenities ({ checkedAmenities, setCheckedAmenities }) {
```

Comment:
Friendly reminder about refactoring those components, make sure that you don't repeat the code 😃